oci/utils_unix.go

```diff
@@ -91,6 +91,9 @@ func getDevices(path, containerPath string) ([]specs.LinuxDevice, error) {
 			}
 			return nil, err
 		}
+		if device.Type == fifoDevice {
+			continue
+		}
```

Comment:
I was looking at https://github.com/opencontainers/runc/pull/2529 to see if there was a mention about this specific check; wouldn't they have  `major == 0`, and because of that exclude by that check? (in that case, these lines wouldn't make a difference

Reply:
```bash
find / -type p | xargs stat -c 'major: %t minor: %T (%n)'
major: 0 minor: 0 (/var/lib/containerd/io.containerd.runtime.v1.linux/moby/72781975f8d0b8d784a99108345438b156d570cfddd83bca7e5017c37621767b/shim.stderr.log)
major: 0 minor: 0 (/var/lib/containerd/io.containerd.runtime.v1.linux/moby/72781975f8d0b8d784a99108345438b156d570cfddd83bca7e5017c37621767b/shim.stdout.log)
major: 0 minor: 0 (/run/docker/containerd/2c2bd632f9e0450a41dc3d57e7824efad4fb00b6e77a804be40246350d3160f4/init-stderr)
major: 0 minor: 0 (/run/docker/containerd/2c2bd632f9e0450a41dc3d57e7824efad4fb00b6e77a804be40246350d3160f4/init-stdout)
major: 0 minor: 0 (/run/containerd/io.containerd.runtime.v2.task/moby/2c2bd632f9e0450a41dc3d57e7824efad4fb00b6e77a804be40246350d3160f4/log)
major: 0 minor: 0 (/run/dmeventd-client)
major: 0 minor: 0 (/run/dmeventd-server)
major: 0 minor: 0 (/run/systemd/ask-password-block/136:0)
major: 0 minor: 0 (/run/systemd/ask-password-block/136:1)
major: 0 minor: 0 (/run/systemd/inhibit/1.ref)
major: 0 minor: 0 (/run/systemd/sessions/13128.ref)
major: 0 minor: 0 (/run/systemd/initctl/fifo)
major: 0 minor: 0 (/run/systemd/inaccessible/fifo)
```

Reply:
nod.. yeah quick check.. I see major/minor in dev.. only applies to S_IFCHR or S_IFBLK not to S_IFREG, S_IFIFO,        or S_IFSOCK types.. so as you say it would've been filtered before as well.

Reply:
I must admit I also had to search for it 😂